### PR TITLE
Updated load-database testscript with VN Resource creation test

### PR DESCRIPTION
**Pull request by sureshkvl**

### CODE REVIEW

**Comment by eonpatapon** [comment]
Usually we prefix the build target in the commit like:

`contrail32.tools.loadDatabase: add VN and LR creation test`

File: test/load-database.nix [hunk 1]
```diff
@@ -56,6 +56,9 @@ let
   testScript = ''
     $machine->waitForOpenPort(8082);
     $machine->waitUntilSucceeds("${pkgs.contrailApiCliWithExtra}/bin/contrail-api-cli ls -l virtual-network | grep -q vn1");
+    $machine->succeed("${pkgs.contrailApiCliWithExtra}/bin/contrail-api-cli --ns contrail_api_cli.provision add-vn --project-fqname default-domain:default-project testvn2");
+    $machine->waitUntilSucceeds("${pkgs.contrailApiCliWithExtra}/bin/contrail-api-cli ls -l routing-instance | grep -q default-domain:default-project:testvn2:testvn2");
```
Comment (eonpatapon) [inline test/load-database.nix]: The RI is created synchronously so you could use `$machine-succeed`

File: test/load-database.nix [hunk 2]
```diff
@@ -56,6 +56,9 @@ let
...
+    $machine->waitUntilSucceeds("${pkgs.contrailApiCliWithExtra}/bin/contrail-api-cli tree routing-instance/default-domain:default-project:testvn2:testvn2 | grep -q route-target");
```
Comment (eonpatapon) [inline test/load-database.nix]: `cat routing-instance/... | grep ...` is enough

File: test/load-database.nix (hunk 1 above)
Comment (sureshkvl) [reply test/load-database.nix]: ok

File: test/load-database.nix (hunk 2 above)
Comment (sureshkvl) [reply test/load-database.nix]: ok

**Pull request merged**